CONTRIBUTING.adoc

```diff
@@ -11,74 +18,81 @@ The best way to install Node is to use {uri-nvm}[nvm] (Node Version Manager).
 
 We recommend using the latest long term support (LTS) release of Node. While you can use other versions of Node, this extension is only tested against active LTS releases.
 
+.Note
 NOTE: If you feel more comfortable with `yarn`, you can use it as an alternative to `npm`
 
 == Building
 
 . Install all the dependencies
 +
- $ npm install
+  $ npm install
```

Comment:
I prefer using as few spaces as possible, so a single space of indentation should do here (and elsewhere).